src/test/java/Lesson10.java

```diff
@@ -0,0 +1,110 @@
+    private static final String SUBMIT_UPLOAD_FILE_BUTTON = "file-submit";
+    private static final String UPLOAD_FILE_NAME = "uploaded-files";
+    private static final String CHECK_TEXT_IN_IFRAME = "tinymce";
+
+    @BeforeMethod
+    public void initMethod() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+    }
+
+    @Test
+    public void contextMenuTest() {
+        //open page http://the-internet.herokuapp.com/context_menu
+        driver.get(HEROCU_APP_URL_CONTEXT_MENU);
+        //find dedicated box on page
+        WebElement whiteBox = driver.findElement(By.xpath(WHITE_BOX));
+        //Click on box by right button
+        Actions rightClick = new Actions(driver);
+        rightClick.contextClick(whiteBox).build().perform();
+        //Set waiter on alert message
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+        //wait when alert is displayed
+        wait.until(ExpectedConditions.alertIsPresent());
+        //get text from alert message
+        String actualResult = driver.switchTo().alert().getText();
+        //close alert message
+        driver.switchTo().alert().dismiss();
+        //Check text on expected
+        Assert.assertEquals(actualResult, "You selected a context menu");
+    }
+
+    @Test
+    public void dynamicControl() {
+        //open page http://the-internet.herokuapp.com/dynamic_controls
+        driver.get(HEROCU_APP_URL_DYNAMIC_CONTROL);
+        //find checkbox
+        driver.findElement(By.xpath(CHECKBOX));
+        //find button "Remove/add", and click it
+        driver.findElement(By.xpath(BUTTON_REMOVE_ADD)).click();
+        //wait when message "It's gone!" is displayed
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(MESSAGE_CHECKBOX_IS_GONE)));
+        // checkbox displayed verification
+        Assert.assertTrue(driver.findElement(By.xpath(MESSAGE_CHECKBOX_IS_GONE)).isDisplayed());
+        //Check input field is disabled
+        Assert.assertFalse(driver.findElement(By.xpath(INPUT)).isEnabled());
+        //find button "Enable" and click it
+        driver.findElement(By.xpath(BUTTON_ENABLE_INPUT)).click();
+        //wait when message "It's enabled!" is displayed
+        wait.until(ExpectedConditions.visibilityOfElementLocated(MESSAGE_INPUT_ENABLED));
+        //chek input field is enabled
+        Assert.assertTrue(driver.findElement(By.xpath(INPUT)).isEnabled());
+    }
+
+    @Test
+    public void uploadFileTest() {
+        //open page http://the-internet.herokuapp.com/upload
+        driver.get(HEROCU_APP_UPLOAD_FILE);
+        //find button "open file", and send absolute path to upload file
+        driver.findElement(By.id(FILE_UPLOAD_BUTTON)).sendKeys("C:\\Users\\Бубята\\IdeaProjects\\HerokuApp\\src\\test\\resources\\Lesson_4.jpg");
```

Comment:
1. Don't use absolute path, I can't run your test because I have no folder C:\\Users\\Бубята\\IdeaProjects and etc.
2. Include your file in PR. I should be able to launch your test in one click.

Reply:
Done!